core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -33,7 +33,7 @@ trait RTS {
   /**
    * Effectfully interprets an `IO`, blocking if necessary to obtain the result.
    */
-  final def tryUnsafePerformIO[E, A](io: IO[E, A]): ExitResult[E, A] = {
+  final def tryUnsafeRun[E, A](io: IO[E, A]): ExitResult[E, A] = {
```

Comment:
This one should be called `unsafeRunSync`.

Reply:
Done!